californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
> Nevertheless the used dependency from Maven does not contain OSGi meta-data as you already noticed. We could either change the dependency to the OSGi versions of that bundles (I really have no clue why they have two versions) or keep it that way.

Do you know the maven coordinates of these?

IMHO we definitely need to have the minimum version in the imports. So, I'd rather try to use OSGi-fied versions of the http client jars but as a fall back we could also add that information manually to the Felix plugin configuration, couldn't we?